### PR TITLE
fix: Rename function for rotation area in LVGL port.

### PR DESCRIPTION
**Pull request by espzav**
# ESP-BSP Pull Request checklist

- [ ] Version of modified component bumped
- [ ] CI passing

# Change description
- Renamed function for rotation area in LVGL port (linking error with master LVGL)

### CODE REVIEW

File: components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c [hunk 1]
```diff
@@ -428,7 +428,7 @@ static void _lvgl_port_transform_monochrome(lv_display_t *display, const lv_area
     }
 }
 
-void lv_display_rotate_area(lv_display_t *disp, lv_area_t *area)
+void lvgl_port_rotate_area(lv_display_t *disp, lv_area_t *area)
```
Comment (tore-espressif) [inline components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c:431]: Other option for consideration:
* We can declare this function as `weak` so the linker can use either this or LVGL's implementation, if it is present

I'm not sure though, whether this is a good idea for this case...

File: components/esp_lvgl_port/idf_component.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: "2.3.0"
+version: "2.3.0~1"
```
Comment (tore-espressif) [inline components/esp_lvgl_port/idf_component.yml:1]: Could we keep the version for now? I expect more changes to esp_lvgl_port soon

**Review by tore-espressif — COMMENTED** [review]
Thanks for the quick fix @espzav !

File: components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c (hunk 1 above)
Comment (espzav) [reply components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c:431]: I suggest, we can keep it like this now.

File: components/esp_lvgl_port/idf_component.yml (hunk 2 above)
Comment (espzav) [reply components/esp_lvgl_port/idf_component.yml]: ok

**Review by tore-espressif — APPROVED** [review]

**Pull request merged**